src/content/structured/get-started/releases-versioning.mdx

```diff
@@ -72,6 +72,40 @@ We consider Beta components acceptable for use in production-ready applications,
 
 We regularly review the stability and reliability of Beta components. When the component meets the criteria it can be transitioned to a stable release.
 
+## Canary components
+
+Canary components are unstable components in our versioning system that are released for testing purposes.
+
+These components should not be used in production apps without understanding the risk that unexpected changes may be made to them, causing changes within the applications using them.
+
+We consider a canary component to be acceptable for moving to Beta once customers have tested its usage and provided enough feedback suggesting they're happy with it.
+
+Canary components can be installed in the same way as normal `@ukic/` packages, but prefixed with canary, e.g. `@ukic/canary-react` or `@ukic/canary-web-components`. You can update to the latest canary version just as you would with normal `@ukic/` packages.
```

Comment:
```suggestion
Canary components can be installed in the same way as normal `@ukic/` packages, but prefixed with canary, e.g. `@ukic/canary-react` or `@ukic/canary-web-components`. 
```

Wondering if the last sentence can be removed and add the command just like it's done on the 'Get the components' pages, so something like ....

```
// Canary Web Components
npm i @ukic/canary-web-components @ukic/fonts

// Canary React
npm i @ukic/canary-react @ukic/fonts
```